Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -14,12 +17,17 @@ public partial class WrapPanel
         [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
         private struct UvMeasure
         {
-            internal static readonly UvMeasure Zero = default(UvMeasure);
+            internal static readonly UvMeasure Zero = default;
```

Comment:
This doesn't need to be stored in an actual field, we can turn this into a getter so that the JIT can see that it just returns `default` and it can inline this call entirely and just zero the target memory without doing any reads 🚀

```suggestion
            internal static UvMeasure Zero => default;
```